frontend/src/scss/global.scss

```diff
@@ -98,3 +98,11 @@ body {
 .d-flow-root {
   display: flow-root;
 }
+
+.d-contents {
+  display: contents;
+}
```

Comment:
Support is not generally bad: https://caniuse.com/mdn-css_properties_display_contents
It just has caveats that need to be considered: https://adrianroselli.com/2022/07/its-mid-2022-and-browsers-mostly-safari-still-break-accessibility-via-display-properties.html
(Generally with display properties)

Reply:
Oh I see, what's the difference between your and my caniuse link then? And do you actually use this class anywhere? Or can you remove it?

Reply:
I think the difference is, wether it works from a layouting perspective or 100% in every area (accessibility, rendering, interop, …). Currently there are some things left to be specified from the csswg/browsers.

I've used while prototyping and thought it would not hurt as it was missing (I've just found out, that `grid`, `inline-grid` are missing as well). I will remove it, as it is not in use currently.